page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/transformer/enums/Condition.java

```diff
@@ -0,0 +1,36 @@
+package ru.sbtqa.tag.pagefactory.transformer.enums;
+
+import cucumber.deps.com.thoughtworks.xstream.annotations.XStreamConverter;
+import cucumber.runtime.CucumberException;
+import ru.sbtqa.tag.pagefactory.transformer.ConditionTransformer;
+
+@XStreamConverter(ConditionTransformer.class)
+public enum Condition {
+
+    NEGATIVE("не"),
+    NEGATIVE_EN("not"),
```

Comment:
It doesn't good to mix different languages in one enum.
Let's split em and implement smart enum interface. All be NEGATIVE but depending on language will be in different implementations. Because if you work w/o cucumber it will confuse user to see NEGATIVE and NEGATIVE_EN in the drop-down.

Reply:
Also I don't see where NEGATIVE ones in tests.

Reply:
так и хотела сделать, но у них завязка на один и тот же метод.
NEGATIVE ты можешь туда передавать и он попадает из шагов. там так же отслеживается, чтобы ничего другого, кроме not и не в  негативном не было